easybuild/easyconfigs/g/Gurobi/Gurobi-9.5.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+name = 'Gurobi'
+version = '9.5.0'
+
+homepage = 'https://www.gurobi.com'
+description = """The Gurobi Optimizer is a state-of-the-art solver for mathematical programming.
+The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern
+architectures and multi-core processors, using the most advanced implementations of the
+latest algorithms."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://packages.gurobi.com/%(version_major_minor)s/']
+sources = ['%(namelower)s%(version)s_linux64.tar.gz']
+patches = ['%(name)s-9.0.1_use-eb-python-gurobi-shell.patch']
+checksums = [
+    'bb542e5a5d1685f6e364eaf05ea6c5572485ffc37a2245169cf3cb40288bc29e',  # gurobi9.5.0_linux64.tar.gz
+    'b4a998182d05f969d1de519f4746ac9e0c6646dd35233231b6ab5963dfa67d01',  # Gurobi-9.0.1_use-eb-python-gurobi-shell.patch
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+]
+
+# remove bundled Python interpreter in favour of the dependency in EB
+postinstallcmds = ['rm %(installdir)s/bin/python*']
+
+# license is not mandatory for installation
+license_file = HOME + '/licenses/%(name)s.lic'
```

Comment:
@boegel this appears to be a bit misleading, given that the build fails without it.

Reply:
@verdurin fixed in b3f588d